### ISSUE TITLE
Uncaught Error: Pane items must be objects. Attempted to add item undefined.

### ISSUE THREAD

**Issue by levity** [leny/atom-pane-manager#3]
Steps to reproduce:
1. Add a new empty pane to the layout using e.g. Cmd-K Cmd-L Cmd-2.
2. Attempt to move an open tab to the new empty pane, not realizing that focus is still on the empty pane, using Cmd-K Cmd-M Cmd-2.

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [pane-manager](https://github.com/leny/atom-pane-manager) package, v0.1.5
### Stack Trace

Uncaught Error: Pane items must be objects. Attempted to add item undefined.

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:311

Error: Pane items must be objects. Attempted to add item undefined.
    at Pane.module.exports.Pane.addItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:311:15)
    at Pane.module.exports.Pane.moveItemToPane (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:427:19)
    at Function.module.exports.PaneUtilities.moveItemTo (/Users/lawrence/.atom/packages/pane-manager/lib/pane-utilities.coffee:73:55)
    at Object.module.exports.moveToPane (/Users/lawrence/.atom/packages/pane-manager/lib/pane-manager.coffee:228:41)
    at atom-workspace.<anonymous> (/Users/lawrence/.atom/packages/pane-manager/lib/pane-manager.coffee:123:24)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
    at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)
```
### Commands

```
     -0:07.0 core:close (atom-text-editor.editor.is-focused)
     -0:04.0 pane-manager:changeLayoutTwo (atom-text-editor.editor.is-focused)
     -0:01.5.0 pane-manager:moveToPaneTwo (atom-pane.pane.active)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-dark-syntax"
    ],
    "disabledPackages": [
      "cursor-count",
      "cursor-count",
      "incremental-search",
      "emmet"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
emmet-simplified, v2.4.2
jshint, v1.3.9
language-ejs, v0.2.0
linter-tidy, v1.0.1
minimap, v4.10.1
pane-manager, v0.1.5
pigments, v0.8.3
tidy-tabs, v0.2.0

# Dev
No dev packages
```

**Comment by levity** [comment]
This is sort of caused by user error, because the focus wasn't on the pane I intended to move from; but of course it would be nice if the attempt did nothing rather than pop up an uncaught error dialog.

It would be even nicer if the command automatically moved the active tab in the last-used non-empty pane. :)

**Comment by leny** [comment]
Since version `1.0`, I can't reproduce your bug, I think its fixed _via_ the JD342's PR.

**Comment by levity** [comment]
In 1.0 it no longer throws an error, but the desired behavior still doesn't happen. JD342's PR is about moving a file to a pane that already exists. My repro steps are about creating a new pane, and then attempting to move a file to it.

**Comment by leny** [comment]
If I understand well, to desired behavior is to get back the focus to the element which has it before the split, right?

**Comment by levity** [comment]
Yes, exactly.

**Comment by leny** [comment]
Ok, thanks.
I'm on it.

**Comment by levity** [comment]
Thank you!

**Comment by leny** [comment]
Fixed with version `1.0.1`.
Was a really dumb typo. I feel terrible for the long delay on this.
